source/fundamentals/indexes.txt

```diff
@@ -128,23 +128,22 @@ To learn more, see  :manual:`Compound Indexes </core/index-compound>`.
 Multikey Indexes (Indexes on Array Fields)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**Multikey indexes** are indexes that improve performance for queries that
-specify ascending or descending indexes on fields that contain an array value.
-You can define a multikey index using the same syntax as a single field or
-compound index.
+**Multikey indexes** are indexes that improve performance for queries on
+fields that contain array values. You can define a multikey index using
+the same syntax as when you create a single field or compound index.
```

Comment:
Suggestion:
I wonder if the second sentence here is necessary since you show the syntax. The only value I can see is if a reader had just read the prior two sections, but then might have to refer back to them to understand what that syntax was. I would probably omit that sentence.